pyaerocom/io/read_eea_aqerep.py

```diff
@@ -0,0 +1,538 @@
+                        if header[idx] != self.VAR_CODE_NAME:
+                            data_dict[header[idx]] = rows[idx]
+                        else:
+                            #extract the EEA var code from the URL noted in the data file
+                            data_dict[header[idx]] = rows[idx].split('/')[-1]
+
+                for idx in file_indexes_to_keep:
+                    # if the data is a time
+                    if idx in time_indexes:
+                        # make the time string ISO compliant so that numpy can directly read it
+                        # this is not very time string forgiving but fast
+                        data_dict[header[idx]][lineidx] = np.datetime64(rows[idx][0:10]+'T'+rows[idx][11:19]+rows[idx][20:])
+                    else:
+                        # data is not a time
+                        # sometimes there's no value in the file. Set that to nan
+                        try:
+                            data_dict[header[idx]][lineidx] = np.float_(rows[idx])
+                        except ValueError:
+                            data_dict[header[idx]][lineidx] = np.nan
+
+                lineidx += 1
+
+        # Empty data object (a dictionary with extended functionality)
+        data_out = StationData()
+        data_out.data_id = self.data_id
+        data_out.dataset_name = self.DATASET_NAME
+        data_out.station_id = data_dict['airqualitystation']
+        data_out.station_name = data_dict['airqualitystation']
+        data_out.filename = filename
+        data_out.instrument_name = self.INSTRUMENT_NAME
+        data_out.country_code = data_dict['countrycode']
+        data_out.ts_type = data_dict['averagingtime'] + 'ly'
+        data_out['variables'] = [aerocom_var_name]
+        data_out['var_info'][aerocom_var_name] = od()
+        data_out['var_info'][aerocom_var_name]['units'] = 'ug m-3'
+        data_out['var_info'][aerocom_var_name]['ts_type'] = 'hourly'
+
+        for key, value in data_dict.items():
+            # adjust the variable name to aerocom standard
+            if key != self.VAR_NAMES_FILE[aerocom_var_name]:
+                data_out[key] = value[:lineidx]
+            else:
+                data_out[aerocom_var_name] = value[:lineidx]
+
+        # just assume hourly data for now
+        time_diff = np.timedelta64(30,'m')
+        data_out['dtime'] = data_dict[self.START_TIME_NAME][:lineidx] + time_diff
+        # convert data vectors to pandas.Series (if attribute
+        # vars_as_series=True)
+        if vars_as_series:
+            data_out[aerocom_var_name] = pd.Series(data_out[aerocom_var_name], index=data_out['dtime'])
+
+        return data_out
+
+
+    def _read_metadata_file(self, filename=None):
+        """Read EEA metadata file
+
+        Parameters
+        ----------
+        filename : str
+            Absolute path to filename to read.
+
+        Returns
+        -------
+        StationData
```

Comment:
```suggestion
        dict
```
?

Reply:
this was actually a copy / paste error. Resolved.